BGL/include/CGAL/boost/graph/parameters_interface.h

```diff
@@ -64,6 +64,7 @@ CGAL_add_named_parameter(nb_points_per_distance_unit_t, nb_points_per_distance_u
 CGAL_add_named_parameter(outward_orientation_t, outward_orientation, outward_orientation)
 CGAL_add_named_parameter(overlap_test_t, overlap_test, do_overlap_test_of_bounded_sides)
 CGAL_add_named_parameter(preserve_genus_t, preserve_genus, preserve_genus)
+CGAL_add_named_parameter(apply_per_connected_component_t, apply_per_connected_component, apply_per_connected_component)
```

Comment:
Still ought to be tested :>

( https://github.com/CGAL/cgal/blob/master/BGL/test/BGL/test_cgal_bgl_named_params.cpp , which is not the most interesting test file ever I will give you that, but there were bugs in `parameters_interface.h` file in the past.)

Reply:
It is tested now.